pkg/graph/resolvers/status_evaluation_strategies.go

```diff
@@ -0,0 +1,143 @@
+package resolvers
+
+import (
+	"time"
+
+	"github.com/cmsgov/mint-app/pkg/shared/utilitytime"
+
+	"github.com/cmsgov/mint-app/pkg/models"
+)
+
+// GetAllStatusEvaluationStrategies returns all status evaluation strategies
```

Comment:
Can you update/add comment on this function that explains the criticality/importance of the order of the strategies? (Mainly that the first matching strategy to evaluate to true is what gets used, so that's why they're in the order they're in)